protostar/protostar_toml/io/protostar_toml_reader.py

```diff
@@ -87,3 +87,14 @@ def _read_if_cache_miss(self) -> Dict[str, Any]:
             self._cache = protostar_toml_flat_dict
 
             return protostar_toml_flat_dict
+
+    @staticmethod
+    def find_protostar_toml_path() -> Optional[Path]:
+        directory_path = Path().resolve()
+        root_path = Path(directory_path.root)
+        while directory_path != root_path:
+            if "protostar.toml" in listdir(directory_path):
+                return directory_path / "protostar.toml"
+
+            directory_path = directory_path.parent
+        return None
```

Comment:
would it make sense to make this function simply global function instead of a static method? as far as I see you are always calling  `ProtostarTOMLReader.find_protostar_toml_path()` which is more verbose than just `find_protostar_toml_path()`